components/automate-workflow-ctl/Gemfile.lock

```diff
@@ -50,70 +50,103 @@ GEM
       mixlib-log (~> 1.3)
       rack (~> 2.0)
       uuidtools (~> 2.1)
-    coderay (1.1.1)
+    coderay (1.1.2)
+    concurrent-ruby (1.1.5)
+    declarative (0.0.10)
+    declarative-option (0.1.0)
     diff-lcs (1.3)
-    docker-api (1.33.6)
-      excon (>= 0.38.0)
-      json
-    elasticsearch (5.0.4)
-      elasticsearch-api (= 5.0.4)
-      elasticsearch-transport (= 5.0.4)
-    elasticsearch-api (5.0.4)
-      multi_json
-    elasticsearch-transport (5.0.4)
-      faraday
+    docker-api (1.34.2)
+      excon (>= 0.47.0)
       multi_json
+    domain_name (0.5.20180417)
+      unf (>= 0.0.5, < 1.0.0)
     erubis (2.7.0)
-    excon (0.57.1)
-    faraday (0.12.1)
+    excon (0.64.0)
+    faraday (0.15.4)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.18)
+    faraday-cookie_jar (0.0.6)
+      faraday (>= 0.7.4)
+      http-cookie (~> 1.0.0)
+    ffi (1.10.0)
     ffi-yajl (2.3.1)
       libyajl2 (~> 1.2)
     fuzzyurl (0.9.0)
+    google-api-client (0.23.9)
+      addressable (~> 2.5, >= 2.5.1)
+      googleauth (>= 0.5, < 0.7.0)
+      httpclient (>= 2.8.1, < 3.0)
+      mime-types (~> 3.0)
+      representable (~> 3.0)
+      retriable (>= 2.0, < 4.0)
+      signet (~> 0.9)
+    googleauth (0.6.7)
+      faraday (~> 0.12)
+      jwt (>= 1.4, < 3.0)
+      memoist (~> 0.16)
+      multi_json (~> 1.11)
+      os (>= 0.9, < 2.0)
+      signet (~> 0.7)
     gssapi (1.2.0)
       ffi (>= 1.0.1)
     gyoku (1.3.1)
       builder (>= 2.1.2)
-    hashie (3.5.6)
-    highline (1.7.8)
+    hashie (3.6.0)
+    highline (1.7.10)
+    http-cookie (1.0.3)
+      domain_name (~> 0.5)
     httpclient (2.8.3)
+    inifile (3.0.0)
     iniparse (1.4.4)
     ipaddress (0.8.3)
-    json (2.1.0)
+    json (2.2.0)
+    jwt (2.1.0)
     libyajl2 (1.2.0)
     little-plugger (1.1.4)
     logging (2.2.2)
       little-plugger (~> 1.1)
       multi_json (~> 1.10)
-    method_source (0.8.2)
-    mixlib-archive (0.4.1)
+    memoist (0.16.0)
+    method_source (0.9.2)
+    mime-types (3.2.2)
+      mime-types-data (~> 3.2015)
+    mime-types-data (3.2019.0331)
+    mixlib-archive (0.4.20)
       mixlib-log
     mixlib-authentication (1.4.2)
     mixlib-cli (1.7.0)
-    mixlib-config (2.2.4)
-    mixlib-install (3.3.1)
+    mixlib-config (2.2.18)
+      tomlrb
```

Comment:
FYI - upgrading maxlib-config from 2.2.4 to 2.2.18 stops it from being able to work with Ruby < v2.2 
https://github.com/chef/mixlib-config/commit/988c13aaecfae5afb7d4961a6181d8e6057a0a71